docs/source/tensors.rst

```diff
@@ -42,6 +42,14 @@ A tensor can be constructed from a Python :class:`list` or sequence using the
      1 -1
     [torch.FloatTensor of size (2,2)]
 
+.. warning::
+
+    :func:`torch.tensor` always copies :attr:`data`. If you have a Tensor
+    ``data`` and want to avoid a copy, use :func:`torch.Tensor.requires_grad_`
```

Comment:
@zou3519 Can you clarify here that users should use `.to` if want to change types/devices, and use these two methods if they want to change `requires_grad`.

Reply:
Yup!

Reply:
I feel like there is a phrase missing here.  What is the user trying to do in each case?